Library/Formula/pgroonga.rb

```diff
@@ -0,0 +1,39 @@
+class Pgroonga < Formula
+  desc "PostgreSQL plugin to use Groonga as index"
+  homepage "https://pgroonga.github.io/"
+  url "http://packages.groonga.org/source/pgroonga/pgroonga-1.0.0.tar.gz"
+  sha256 "854d66bdc79e7cfceb29ae8706556a3ad1c8b32ac4535939d296966e3b19fd3e"
+
+  depends_on "pkg-config" => :build
+  depends_on "groonga"
+  depends_on "postgresql"
+
+  def install
+    # To find HOMEBREW_PREFIX/bin/pg_config
+    ENV.prepend_path "PATH", HOMEBREW_PREFIX/"bin"
```

Comment:
The problem for using one mega formula is it doesn't scale well considering so many plugins out there. At the same time, users won't get upgrading notice for each plugin formulae. And final but not least, I feel a little uncomfortable to mix the package with multiple different sources and authors. 

About allowing to prefix, I would really like to avoid it. It's very fragile. For example, whenever Postgres got upgraded or user doing reinstall, all of plugins have to been manually reinstalled. Deep down, current problem has little thing to do with sandbox, as it has already been broken in too many ways.